tests/integration-tests/tests/cfn-init/test_cfn_init.py

```diff
@@ -23,7 +23,7 @@
 
 @pytest.mark.regions(["eu-central-1"])
 @pytest.mark.instances(["c5.xlarge"])
-@pytest.mark.schedulers(["slurm", "sge"])
+@pytest.mark.schedulers(["slurm"])
```

Comment:
probably not part of this PR but we should get rid of the regions, instances, schedules. oss annotations since now we use the config file